### DISCUSSION TITLE
router helper

### DISCUSSION THREAD

**Discussion by saeedvaziry** [laravel/framework#40837] [Ideas]
Is there any chance to have a helper named `router()` to use the `Route` facade?

This helper will return an instance of `Route` facade

We can use it like

```php
router()->get('/', function() {
    return 'Hi 👋 ';
});
```

Instead of this

```php
use Illuminate\Support\Facades\Route;

Route::get('/', function() {
    return 'Hi 👋 ';
});
```

**Comment by browner12** [comment]
What's the use case?

**Reply by saeedvaziry** [reply]
Just for making it simpler to use
